content/en/docs/tasks/administer-cluster/kubelet-in-userns.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Running Kubernetes node components as a non-root user (`KubeletInUserNamespace`)
+content_type: task
+---
+
+{{< feature-state for_k8s_version="v1.22" state="alpha" >}}
+
+This document describes how to run Kubernetes node components such as kubelet, CRI, OCI, and CNI
+without root privileges, by using {{< glossary_tooltip text="a user namespace" term_id="userns" >}}
+and `KubeletInUserNamespace` feature gate.
```

Comment:
Move this detail to the prerequisites section, and hyperlink the text “feature gate” to https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/

Reply:
updated